docs/guides/upgrade_paths_for_neon_os_v2.md

```diff
@@ -0,0 +1,19 @@
+# Ugrade Paths for Neon OS Version 2.0 on the Mark II
+Describes your basic options for upgrading your Neon OS to V2.0.
+
+## Options
+1. Make a version 2.0 boot drive on your Mark II using the "Make Bootable Media" skill.
+2. Make a version 2.0 boot drive using a computer.
+3. Order a free USB with the image on it from Neon AI with coupon code "UPGRADE". Our square site is https://neonai.square.site/
+4. Order an SSD from our Square site for $29.99 with coupon code "UPGRADE". The site is https://neonai.square.site/
+5. Visit the upgrade page on our website at https://neon.ai/V2 for written directions, how-to videos, and open tech support calls. That's https://neon.ai/V2
+
+- To hear more detailed directions, you can ask Neon for:
+    - help making bootable media
+    - help making a boot drive using your computer
```

Comment:
This won't be implemented in the near future so it should probably not go in docs yet

Reply:
Ah. I am confused then - I saw you had one other set of directions up for a template, so thought there would be a few different "help" choices that it could read out.

Reply:
For now it's just written content, but formatted so that we can implement that skill/intent to find/read help content